javaslang-benchmark/src/test/java/javaslang/BenchmarkPerformanceReporter.java

```diff
@@ -328,18 +324,18 @@ private void printTargetInDenominator() {
             System.out.println(String.format("\nRatios <alternative_impl> / %s", targetImplementation));
             System.out.println(ratioHeaderDenominator());
             for (String targetImpl : targetImplementations) {
-                for (Tuple2<String, List<TestExecution>> execution : resultsByKey) {
+                for (Tuple2<String, Array<TestExecution>> execution : resultsByKey) {
                     printRatioForBaseType(targetImpl, execution._2,
                             (baseImpl, alternativeImpl) -> padRight(String.format("%s/%s", alternativeImpl, baseImpl), ratioSize),
                             (baseExec, alternativeExec) -> calculateRatios(alternativeExec, baseExec));
                 }
             }
         }
 
-        private void printRatioForBaseType(String baseType, List<TestExecution> testExecutions,
+        private void printRatioForBaseType(String baseType, Array<TestExecution> testExecutions,
                                            Function2<String, String, String> ratioNamePrinter,
```

Comment:
In src/main we use `BiFunction` to widen the type range (`Function2 extends BiFunction`). But it is ok here!